pkg/volume/vsphere_volume/attacher.go

```diff
@@ -90,22 +90,22 @@ func (attacher *vsphereVMDKAttacher) Attach(spec *volume.Spec, nodeName types.No
 	// succeeds in that case, so no need to do that separately.
 	diskUUID, err := attacher.vsphereVolumes.AttachDisk(volumeSource.VolumePath, volumeSource.StoragePolicyName, nodeName)
 	if err != nil {
-		klog.Errorf("Error attaching volume %q to node %q: %+v", volumeSource.VolumePath, nodeName, err)
+		klog.ErrorS(err, "Error attaching volume to node", "volumePath", volumeSource.VolumePath, "nodeName", nodeName)
 		return "", err
 	}
 
 	return filepath.Join(diskByIDPath, diskSCSIPrefix+diskUUID), nil
 }
 
 func (attacher *vsphereVMDKAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.NodeName) (map[*volume.Spec]bool, error) {
-	klog.Warningf("Attacher.VolumesAreAttached called for node %q - Please use BulkVerifyVolumes for vSphere", nodeName)
+	klog.InfoS("Attacher.VolumesAreAttached called for node - Please use BulkVerifyVolumes for vSphere", "nodeName", nodeName)
 	volumeNodeMap := map[types.NodeName][]*volume.Spec{
 		nodeName: specs,
 	}
 	nodeVolumesResult := make(map[*volume.Spec]bool)
 	nodesVerificationMap, err := attacher.BulkVerifyVolumes(volumeNodeMap)
 	if err != nil {
-		klog.Errorf("Attacher.VolumesAreAttached - error checking volumes for node %q with %v", nodeName, err)
+		klog.ErrorS(err, "Attacher.VolumesAreAttached - error checking volumes for node", "nodeName", nodeName)
```

Comment:
```suggestion
		klog.ErrorS(err, "Attacher.VolumesAreAttached - error checking volumes for node", "node", klog.KRef("", nodeName))
```

Reply:
Thanks for your suggestion, we replace `nodeName` with `klog.KRef("", string(nodeName))`, for that variable  `nodeName` belongs to `types.NodeName`.